config/core_pools_whitelist.json

```diff
@@ -1,10 +1,8 @@
 {
   "mainnet": {
     "0xxxx": "WHITELIST-ME",
-    "0x1e19cf2d73a72ef1332c882f20534b6519be0276000200000000000000000112": "B-rETH-STABLE",
     "0xf7a826d47c8e02835d94fb0aa40f0cc9505cb1340002000000000000000005e0": "ECLP-wstETH-cbETH",
```

Comment:
We should figure out autoselection for Gyro pools.  It's maybe a little tricky because of how the curves can be shaped, but in general I think a gyro pool with 1 rate provider should default to core pool.

Reply:
yea so gyro pools appear to not have a yield share in the subgraph, that is why they arent being captured currently.

im in the process of figuring out if we can just include all gyro pools or that there is a different way of making sure they have a yield fee.

Reply:
4cfff67